### PR TITLE
Fix MP browser country name overflow

### LINKED ISSUE #10473: Long country names will overlap the server status field.
I originally decided not to use `WidgetUtils.TruncateText` because I assumed that I had given enough room for all the country names.  The GeoIP country database disagrees.

screen shot 2016-01-10 at 19 39 36

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/29751943-long-country-names-will-overlap-the-server-status-field?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by pchote**
Fixes #10473.

### CODE REVIEW

File: OpenRA.Mods.Common/Widgets/Logic/MultiplayerLogic.cs [hunk 1]
```diff
@@ -402,11 +402,22 @@ void RefreshServerListInner(IEnumerable<GameServer> games)
...
 
+					if (title != null)
+					{
+						var font = Game.Renderer.Fonts[title.Font];
+						var label = WidgetUtils.TruncateText(game.Name, title.Bounds.Width, font);
+						title.GetText = () => label;
+						title.GetColor = () => canJoin ? title.TextColor : incompatibleGameColor;
+					}
+
+
```
Comment (Phrohdoh) [inline OpenRA.Mods.Common/Widgets/Logic/MultiplayerLogic.cs]: Extra newline.

**Comment by pchote** [comment]
Fixed.

**Comment by Mailaender** [comment]
Works as promised. :white_check_mark:

**Comment by RoosterDragon** [comment]
:+1:

**Comment by RoosterDragon** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/b120191e6a0ae56871ac90bdad6c27117cb3bf3e)

**Pull request merged**